WordPress/Classes/ViewRelated/Aztec/Stock Photos/StockPhotosPlaceholder.swift

```diff
@@ -24,6 +28,13 @@ final class StockPhotosPlaceholder: WPNoResultsView {
     }
 
     private func configureSubtitle() {
-        messageText = .freePhotosPlaceholderSubtitle
+        attributedMessageText = createStringWithLinkAttributes(from: .freePhotosPlaceholderSubtitle)
+    }
+
+    private func createStringWithLinkAttributes(from subtitle: String) -> NSAttributedString {
+        let htmlTaggedLink = "<a href=\"\(companyUrl)\">\(companyName)</a>"
+        let htmlTaggedText = subtitle.replacingOccurrences(of: companyName, with: htmlTaggedLink)
+
+        return NSAttributedString.attributedStringWithHTML(htmlTaggedText, attributes: nil)
```

Comment:
@frosty is right, this looks a lot cleaner by using `attributedStringWithHTML`

👍